npm-shrinkwrap.json

```diff
@@ -498,7 +498,7 @@
     },
     "clone": {
       "version": "2.1.2",
-      "resolved": "https://registry.npmjs.org/clone/-/clone-2.1.2.tgz",
+      "resolved": "https://purecloud.jfrog.io/purecloud/api/npm/inin-internal-npm/clone/-/clone-2.1.2.tgz",
```

Comment:
It would be best to undo the changes to this file. We want to keep it using the public registry to avoid people outside of the organization having issues building.